libmscore/style.cpp

```diff
@@ -406,9 +406,9 @@ static const StyleType styleTypes[] {
       { Sid::ottava8VAText,           "ottava8VAText",           QString("<sym>ottavaAlta</sym>") },
       { Sid::ottava8VBText,           "ottava8VBText",           QString("<sym>ottavaBassaBa</sym>") },
```

Comment:
Shouldn't this be ottavaBassaVb?

Reply:
Why?

Reply:
Well, referring back to _Behind Bars_, it seems all of these are already correct after all.

![img_77517adac324-1](https://user-images.githubusercontent.com/35512972/50790954-60c9ba80-128e-11e9-896a-60027690d576.jpeg)

Reply:
Hmm, OK so I guess we can rest that case.
Only other option would be to provide multiple choices for those